### ISSUE TITLE
Keyboard mappings unusable

### ISSUE THREAD

**Issue by Crazee** [nextcloud/vm#894]
## Steps to reproduce
1. Create VM from .ova
2. Try to configure SSL via Let's Encrypt
3. Can't input a valid email address because there is no apparent mapping for the @ character

## Expected behaviour
Tell us what should happen
Should be able to get a working keyboard standard US keyboard (PC or MacBook Pro)

## Actual behaviour
Tell us what happens instead
Can't input the '@' character to can't input a valid email address

## Server configuration

**Nextcloud server version:** (see your admin page)

**Server OS** (Ubuntu server is default)

**How did you install the VM?** (Scripted install from master **OR** Released version)
released.

## Network
**Do you use DHCP?**
yes (for now)

**Is port 80 and/or 443 open?**
yes, yes

#### Log file (/mnt/ncdata/nextcloud.log)
```
Insert your nextcloud.log file here
```

#### Installation log (command output)
```
Insert the CLI output here
```

**Comment by enoch85** [comment]
Hi!

Please read the documentation, the keyboard mapping is in Swedish: docs.hanssonit.se/s/W6fMouPiqQz3_Mog/virtual-machines-vm/d/W6fMquPiqQz3_Moi/nextcloud-vm?currentPageId=W6ylHOPiqQz3_MpT

This is already fixed in the future version of the VM (you will be able to choose and force the selection), so nothing to do here really.

**Comment by Crazee** [comment]
It asks you to pick a language and keyboard mapping type and apparently ignores them.  Have no idea how to type a '@' on a Swedish keyboard.

**Comment by enoch85** [comment]
It's documented in the link I sent here in this issue.

It's also fixed now and will be released in 16.0.4.
